sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+/**
+ * Represent the operation user was performing when the error happened.
+ */
+public enum ServiceBusErrorSource {
```

Comment:
We need an enum for user code. Also, this must be ExpandableStringEnum so that we can add more sources later.